retrofit/src/main/kotlin/com/badoo/reaktive/retrofit/ReaktiveCallAdapter.kt

```diff
@@ -0,0 +1,36 @@
+package com.badoo.reaktive.retrofit
+
+import com.badoo.reaktive.observable.Observable
+import com.badoo.reaktive.observable.asCompletable
+import com.badoo.reaktive.observable.firstOrComplete
+import com.badoo.reaktive.observable.firstOrError
+import retrofit2.Call
+import retrofit2.CallAdapter
+import java.lang.reflect.Type
+
+internal class ReaktiveCallAdapter(
+        private val responseType: Type,
+        private val isBody: Boolean,
```

Comment:
This set of flags smell, it is possible to specify all of them or any combination. Enum?